src/main/preload.js

```diff
@@ -0,0 +1,29 @@
+const { contextBridge, ipcRenderer } = require('electron');
+
+contextBridge.exposeInMainWorld('electron', {
+  ipcRenderer: {
+    send: ipcRenderer.send,
+    sendSync: ipcRenderer.sendSync,
+    /**
+     * You are still able to use ipcRenderer in a renderer process with contextIsolation set to true.
+     * The contextBridge is what you want to use, although there is a current bug that is preventing from you calling ipcRenderer.on in a renderer process;
```

Comment:
I suggest changing or removing the comment here because considering``ipcRenderer.on`` as a bug is not really an accurate statement and can be misleading as stated by the maintainer of electron on [this](https://github.com/electron/electron/issues/21437#issuecomment-573522360) discussion.